tests/system/plugin-jar.test.ts

```diff
@@ -6,32 +6,17 @@ import { legacyPlugin } from '@snyk/cli-interface';
 
 const testsPath = path.join(__dirname, '..');
 const fixturesPath = path.join(testsPath, 'fixtures');
-const jarsPath = path.join(fixturesPath, 'jars');
 const badPath = path.join(fixturesPath, 'bad');
-const goodAndBadPath = path.join(fixturesPath, 'good-and-bad');
-const jarWrongPkgNamePath = path.join(fixturesPath, 'jar-wrong-package-name');
-const springCorePath = path.join(fixturesPath, 'spring-core');
-const springCoreJar = 'spring-core-5.1.8.RELEASE.jar';
 
-test('inspect with spring-core jar file', async (t) => {
-  const result = await plugin.inspect(springCorePath, springCoreJar);
-  if (legacyPlugin.isMultiResult(result)) {
-    return t.fail('expected single inspect result');
-  }
-  const expected = await readFixtureJSON('spring-core', 'expected.json');
-  // result.metadata depends on platform, so no fixture can be provided
-  t.ok(
-    result!.plugin!.meta!.versionBuildInfo!.metaBuildVersion!.javaVersion,
-    'should contain javaVersion key',
-  );
-  t.ok(
-    result!.plugin!.meta!.versionBuildInfo!.metaBuildVersion!.mavenVersion,
-    'should contain mavenVersion key',
-  );
-  // therefore, only independent objects are compared
-  delete result.plugin.meta;
-  t.same(result, expected, 'should return expected result');
-});
+test('inspect with spring-core jar file', async (t) =>
```

Comment:
shall we move to jest while in here?

Reply:
nah, the only reason we want it is because off bad IDE integration
because we have multiple projects with tap, let's try to find a way to make it work better